modules/manage/partials/whole-cluster-restore.adoc

```diff
@@ -0,0 +1,176 @@
+ifdef::env-kubernetes[. <<Verify that the cluster restore is complete>>.]
+
+=== Prerequisites
+
+You must have the following:
+
+- Tiered Storage enabled on the source cluster.
+- Physical or virtual machines on which to deploy the target cluster.
+
+=== Limitations
+
+- Whole cluster restore supports only one source cluster. It is not possible to consolidate multiple clusters onto the target cluster.
+
+- If a duplicate cluster configuration is found in the target cluster, it will be overwritten by the restore.
+
+- The target cluster should not contain user-managed or application-managed topic data, schemas, users, ACLs, or ongoing transactions.
+
+=== Start a target cluster
+
+ifdef::env-kubernetes[]
+
+Deploy the target Redpanda cluster.
+
+[tabs]
+======
+Helm + Operator::
++
+--
+.`redpanda-cluster.yaml`
+[,yaml]
+----
+apiVersion: cluster.redpanda.com/v1alpha1
+kind: Redpanda
+metadata:
+  name: redpanda
+spec:
+  chartRef: {}
+  clusterSpec:
+    storage:
+      tiered:
+        <tiered-storage-settings>
+    config:
+      cluster:
+        cloud_storage_attempt_cluster_recovery_on_bootstrap: true
+----
+
+```bash
+kubectl apply -f redpanda-cluster.yaml --namespace <namespace>
+```
+--
+Helm::
++
+--
+[tabs]
+====
+--values::
++
+.`cluster-restore.yaml`
+[,yaml]
+----
+storage:
+  tiered:
+    <tiered-storage-settings>
+config:
+  cluster:
+    cloud_storage_attempt_cluster_recovery_on_bootstrap: true
+----
++
+```bash
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+--values cluster-restore.yaml
+```
+
+--set::
++
+```bash
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+  --set storage.tiered.<tiered-storage-settings> \
+  --set config.cluster.cloud_storage_attempt_cluster_recovery_on_bootstrap=true
+```
+====
+--
+======
+
+- `storage.tiered`: Make sure to configure the target cluster with the same Tiered Storage settings as the failed source cluster.
+- `config.cluster.cloud_storage_attempt_cluster_recovery_on_bootstrap`: Automate cluster restore in Kubernetes. Setting to `true` is recommended when using an automated method for deployment. When bootstrapping a cluster with a given bucket, make sure that any previous cluster using the bucket is fully destroyed; otherwise Tiered Storage subsystems may interfere with each other.
+endif::[]
+
+ifndef::env-kubernetes[]
+For self-hosted, follow the steps to xref:deploy:deployment-option/self-hosted/manual/index.adoc[deploy a new cluster].
+
+For Redpanda Cloud, follow the steps to xref:deploy:deployment-option/cloud/index.adoc[provision a new cloud cluster].
+Cloud account credentials (such as secrets or access keys) are not included in backed up metadata. You must manually configure these credentials for the new cluster.
+
+NOTE: Make sure to configure the target cluster with the same Tiered Storage settings as the failed source cluster.
+endif::[]
+
+=== Manage metadata uploads
```

Comment:
@andrwng @JakeSCahill I ended up moving this per Andrew's suggestion, and made a change to the heading as well. Feel free to update if it makes more sense to stick to "Manage metadata uploads" or even change it entirely

Reply:
@andrwng so should this section be part of the Tiered Storage topic so that users can know when configuring TS that if they ever need to do a cluster restore, they should configure these settings?

Then in the prereqs of the cluster restore topic, we can link to that section?

Reply:
Hmm good question. I wouldn't be against it, though it'd be worth mentioning the kind of metadata that is uploaded. The above list is fine, but there are a few details more relevant to the restore in it that ought to be trimmed down if we duplicate that info in another section.

Reply:
@JakeSCahill @andrwng Making sure I'm on the same page: isn't whole cluster restore already available to TS users without having to manually preconfigure these three settings? Since `enable_cluster_metadata_upload_loop` is set to `true` by default, and the other two properties are set with default values as well, then those properties don't have to be touched until users need to tweak the way whole cluster restore works for them. Am I on the right track? 

In that case could we consider flipping what Jake mentioned -- place a little blurb in a suitable place in the TS doc instead, and from there link out to this whole cluster restore doc?

Reply:
If you're restoring from a failed cluster. I assume you can't tweak those values, so they'd need to be considered when setting up TS. @andrwng is that right?

Reply:
>In that case could we consider flipping what Jake mentioned -- place a little blurb in a suitable place in the TS doc instead, and from there link out to this whole cluster restore doc?

This sounds reasonable to me too, but I don't have a strong opinion either way for this, though I think I lean towards this because metadata recovery feels a bit different from what I think of when I think of tiered storage (it's data, not metadata, though not sure if that is how a customer would think about it).

>If you're restoring from a failed cluster. I assume you can't tweak those values, so they'd need to be considered when setting up TS. @andrwng is that right?

That's right. In general, a customer who wants to use cluster restore wouldn't touch the configs. The configs are prerequisites to running through a cluster restore.

Reply:
Also, I think I saw it before but I don't anymore: we should also mention that cloud storage should be configured on the cluster _before_ running through the restore command.